src/_content/posts/introducing-keynote-mario-munoz.md

```diff
@@ -0,0 +1,59 @@
+---
+author: DjangoCon US Communications Team
+category: General
+date: 2024-08-01 06:00:00
+cover:
+    url: /assets/img/blog/mario_munoz.jpg
+    alt: "Mario Munoz"
+layout: post
+title: "Announcing Our Keynotes: Mario Munoz"
+---
+
+### We're honored to announce our seecond keynote speaker for DjangoCon US 2024: Mario Munoz!
+
+We had the opportunity to sit down with Mario and ask her a few questions about his work, his upcoming talk, and what shehee's looking forward to at DjangoCon US 2024.
```

Comment:
```suggestion
We had the opportunity to sit down with Mario and ask his a few questions about his work, his upcoming talk, and what he's looking forward to at DjangoCon US 2024.
```

Reply:
jinx

Reply:
Yeah, but I bet you didn't first change `her` to `his`!

Reply:
you are indeed correct